tensorboard/plugins/custom_scalar/tf_custom_scalar_dashboard/tf-custom-scalar-dashboard.html

```diff
@@ -380,6 +382,9 @@ <h3>The custom scalars dashboard is inactive.</h3>
         const pane = event.target;
         if (pane.opened) {
           this._openedCategories[pane.category.name] = true;
+          Array.from(this._getCharts())
```

Comment:
Can you add a comment indicating why it's necessary to reload the charts here?  Basically just a summary of the commit description.

Reply:
We now use tf-category-paginated-view so no longer require this change. Removed it.